.gitignore

```diff
@@ -1,10 +1,40 @@
-target
+.gradle
```

Comment:
This is mostly the file from the Gradle 2.x project with a few things incorporated from the Gradle 1.x project, most notably the entire `.idea` directory.